sdk/api-learn/Azure.Learn.AppConfig/src/autorest.md

```diff
@@ -0,0 +1,8 @@
+# Generated code configuration
+
+Run `dotnet msbuild /t:GenerateCode` to generate code.
+
+``` yaml
+input-file:
+    -  $(this-folder)/swagger/appconfiguration.json
```

Comment:
@tg-msft - why is this necessary?

Reply:
My thinking was it'd be useful for anyone unfortunate enough to stumble into this file?  And I'm talking more _"link to the swagger part of the training"_ rather than _"add ANOTHER copy of everything swagger related here."_  (And we should do the same to Azure.Template once we can link to parts of the training.)

Reply:
https://github.com/Azure/azure-sdk-pr/issues/329